### PR TITLE
Dont use dependencies

### PR DESCRIPTION
**Pull request by jschueller**
Else dependencies tied to a vc version might be pulled

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -16,29 +16,29 @@ source:
...
+    - bzip2 1.0.*  # [unix]
+    - curl >=7.44.0,<8  # [unix]
+    - expat  # [unix]
     - ncurses 5.9*  # [unix]
-    - xz 5.2.*
-    - zlib 1.2.11
+    - xz 5.2.*  # [unix]
+    - zlib 1.2.11  # [unix]
     - libuv  # [unix]
     - rhash  # [unix]
```
Comment (isuruf) [inline recipe/meta.yaml]: Add vs2015_runtime here

**Comment by jakirkham** [comment]
Was about to say... Thanks for getting on top of this. Glad to see we will be building `cmake` from source on Windows.

**Comment by jakirkham** [comment]
Thanks @jschueller. Maybe we can look at putting them in a subdirectory or statically linking them or something.

**Comment by isuruf** [comment]
With Conda build 3, we can use dependencies without a problem

**Comment by jakirkham** [comment]
Possibly. Depends what all we include in `host`. If there are `vc` conflicts in that listing, we still could have some issues.

**Pull request merged**